iceoryx_posh/test/moduletests/test_mepoo_shared_chunk.cpp

```diff
@@ -59,134 +61,126 @@ class SharedChunk_Test : public Test
+    EXPECT_THAT(sut.getChunkHeader(), Eq(nullptr));
 }
 
-TEST_F(SharedChunk_Test, MoveCTor)
+TEST_F(SharedChunk_Test, VerifyCopyConstructorOfSharedChunk)
 {
-    {
-        iox::mepoo::SharedChunk sut2(std::move(sut));
-    }
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(0));
-    EXPECT_THAT(chunkMgmtPool.getUsedChunks(), Eq(0));
+    SharedChunk sut1(sut);
+
+    EXPECT_EQ((sut1.getChunkHeader())->chunkSize(), (sut.getChunkHeader())->chunkSize());
 }
 
-TEST_F(SharedChunk_Test, CopyAssignment)
+TEST_F(SharedChunk_Test, VerifyMoveConstructorOfSharedChunk)
 {
-    iox::mepoo::SharedChunk sut2(GetChunkManagement(mempool.getChunk()));
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(2));
-    sut2 = sut;
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(1));
+    SharedChunk sut1(chunkManagement);
+    ChunkHeader* header = sut1.getChunkHeader();
+
+    SharedChunk sut2(std::move(sut1));
+
+    ASSERT_EQ(sut1.getChunkHeader(), nullptr);
+    ASSERT_EQ(sut2.getChunkHeader(), header);
+    EXPECT_EQ((sut2.getChunkHeader())->chunkSize(), (sizeof(ChunkHeader) + USER_PAYLOAD_SIZE));
 }
 
-TEST_F(SharedChunk_Test, CopyAssignmentFromPointer)
+TEST_F(SharedChunk_Test, VerifiyCopyAssigmentWithSharedChunk)
 {
-    iox::mepoo::SharedChunk sut2(GetChunkManagement(mempool.getChunk()));
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(2));
-    sut2 = nullptr;
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(1));
+    SharedChunk sut1;
+
+    sut1 = sut;
+
+    EXPECT_EQ((sut1.getChunkHeader())->chunkSize(), (sut.getChunkHeader())->chunkSize());
 }
 
-TEST_F(SharedChunk_Test, MoveAssignment)
+TEST_F(SharedChunk_Test, VerifiyMoveAssigmentForSharedChunk)
 {
-    {
-        iox::mepoo::SharedChunk sut2(GetChunkManagement(mempool.getChunk()));
-        EXPECT_THAT(mempool.getUsedChunks(), Eq(2));
-        sut2 = std::move(sut);
-        EXPECT_THAT(mempool.getUsedChunks(), Eq(1));
-    }
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(0));
+    SharedChunk sut1(chunkManagement);
+    SharedChunk sut2;
+    ChunkHeader* header = sut1.getChunkHeader();
+
+    sut2 = std::move(sut1);
+
+    ASSERT_EQ(sut1.getChunkHeader(), nullptr);
+    ASSERT_EQ(sut2.getChunkHeader(), header);
+    EXPECT_EQ((sut2.getChunkHeader())->chunkSize(), (sizeof(ChunkHeader) + USER_PAYLOAD_SIZE));
 }
 
-TEST_F(SharedChunk_Test, DestructorDefaultCTor)
+TEST_F(SharedChunk_Test, CompareWithSameMemoryChunkComparesToUserPayload)
 {
-    {
-        iox::mepoo::SharedChunk sut2(GetChunkManagement(mempool.getChunk()));
-        EXPECT_THAT(mempool.getUsedChunks(), Eq(2));
-    }
-    EXPECT_THAT(mempool.getUsedChunks(), Eq(1));
+    EXPECT_THAT(sut == sut.getUserPayload(), Eq(true));
 }
 
-TEST_F(SharedChunk_Test, getChunkHeaderWhenInvalid)
+TEST_F(SharedChunk_Test, GetChunkHeaderMethodReturnsNullPointerWhenSharedChunkObjectIsInitialisedWithNullPointer)
 {
-    SharedChunk sut2(nullptr);
-    EXPECT_THAT(sut2.getChunkHeader(), Eq(nullptr));
+    SharedChunk sut(nullptr);
```

Comment:
Wouldn't be ` SharedChunk sut;` sufficient? `m_chunkManagement` is a nullptr by default.
Same for the following tests.